src/emulator/shader/src/translator/alu.cpp

```diff
@@ -127,6 +127,7 @@ bool USSETranslatorVisitor::vmad(
     spv::Id vsrc2 = load(inst.opr.src2, write_mask, 0);
 
     if (vsrc0 == spv::NoResult || vsrc1 == spv::NoResult || vsrc2 == spv::NoResult) {
+        LOG_ERROR("Source Not Loaded!(vsrc0: {}, vsr1: {}, vsrc2: {})", vsrc0, vsrc1, vsrc2);
```

Comment:
```suggestion
        LOG_ERROR("Source not loaded (vsrc0: {}, vsr1: {}, vsrc2: {})", vsrc0, vsrc1, vsrc2);
```
same for the other messages, only capitalize the first word

Reply:
ah yes sorry